crawler/src/main/resources/application.conf

```diff
@@ -0,0 +1,27 @@
+akka {
+  quartz {
+    defualtTimezone = "Europe/Warsaw"
+
+    schedules {
+      EverydayMidnight {
+        description = "Everyday at midnight"
+        expression = "0 0 0 * * ?"
+      }
+    }
+  }
+}
+
+bootstrap {
+  servers = "localhost:9092"
```

Comment:
This should be `akka.kafka.producer.kafka-clients.bootstrap.servers`, not just `bootstrap.servers`.